src/EnergyPlus/HeatRecovery.cc

```diff
@@ -1472,352 +1249,310 @@ namespace HeatRecovery {
+                Z = CMin0 / CMax0;
+
+                CalculateNTUfromEpsAndZ(state, NTU0, ErrStat, Z, this->FlowArr, Eps0);
+
+                switch (ErrStat) {
+                case CalculateNTUBoundsErrors::NoError:
+                    break; // great!
+                case CalculateNTUBoundsErrors::MassFlowRatio:
+                    FatalError = true;
+                    ShowSevereError(state, "In the HeatExchanger:AirToAir:FlatPlate component " + this->Name);
+                    ShowContinueError(state, "  the mass flow ratio is out of bounds");
+                    ShowContinueError(state, format("The mass flow ratio is (Min_Mass_Flow_Rate / Max_Mass_Flow_Rate) = {:.2R}", Z));
+                    ShowContinueError(state, "The mass flow ratio should be >= 0.0 and <= 1.0");
+                    ShowContinueError(state,
+                                      format("Min_Mass_Flow_Rate = {:.2R} [air density] * {:.1R} [Min_Vol_Flow_Rate]",
+                                             RhoAir,
+                                             min(this->NomSupAirVolFlow, this->NomSecAirVolFlow)));
+                    ShowContinueError(state,
+                                      format("Max_Mass_Flow_Rate = {:.2R} [air density] * {:.1R} [Max_Vol_Flow_Rate]",
+                                             RhoAir,
+                                             max(this->NomSupAirVolFlow, this->NomSecAirVolFlow)));
+                    break;
+                case CalculateNTUBoundsErrors::NominalEffectiveness1:
+                    FatalError = true;
+                    ShowSevereError(state, "In the HeatExchanger:AirToAir:FlatPlate component " + this->Name);
+                    ShowContinueError(state, "  the calculated nominal effectiveness is out of bounds");
+                    ShowContinueError(state, format("The effectiveness is {:.3R}", Eps0));
+                    ShowContinueError(state, format("The effectiveness should be >= 0.0 and <= {:.3R}", 1.0 / (1.0 + Z)));
+                    ShowContinueError(state,
+                                      "Eff = (Nom_Sup_Mass_Flow_Rate/Min_Mass_Flow_Rate)*(T_nom_sup_out-T_nom_sup_in)/(T_nom_sec_in-T_nom_sup_in)");
+                    ShowContinueError(state, "The temperatures are user inputs. The mass flow rates are user input volume flow rates");
+                    ShowContinueError(state, format("  times the density of air [{:.2R} kg/m3]", RhoAir));
+                    ShowContinueError(state, "Change these inputs to obtain a physically realizable heat exchanger effectiveness");
+                    break;
+                case CalculateNTUBoundsErrors::NominalEffectiveness2:
+                    FatalError = true;
+                    ShowSevereError(state, "In the HeatExchanger:AirToAir:FlatPlate component " + this->Name);
+                    ShowContinueError(state, "  the calculated nominal effectiveness is out of bounds");
```

Comment:
What is the greatest number of consecutive `ShowContinueError` in EnergyPlus? Is it this one? 7?

Reply:
If I am unable to inject humor into my work, I just don't do it as well. I need it for my own sanity and productivity.